recursion/src/main/java/Factorial.java

```diff
@@ -1,6 +1,12 @@
 public class Factorial {
 
     int of(int base) {
-        throw new UnsupportedOperationException();
+        if (base <= 0) {
```

Comment:
Uncovered statement

Reply:
I'll name new PR to fix tests

Reply:
done - https://github.com/PaulFrmBrn/udemy-ds-and-algo-course/pull/7

Reply:
this branch was also rebased